components/service/glean/src/test/java/mozilla/components/service/glean/EventMetricTypeTest.kt

```diff
@@ -30,9 +29,7 @@ class EventMetricTypeTest {
 
     @Before
     fun setUp() {
-        Glean.initialized = true
-        Glean.applicationId = "test"
-        Glean.configuration = Configuration()
+        resetGlean()
```

Comment:
I just filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1527665) about cleaning up all of these `@Before` functions.  I guess we were thinking alike!

Reply:
I went ahead and cleaned them up here (the ones I could find). I hope that's okay.  If it is, we can point that bug at this PR and close it when it lands.